about.html

```diff
@@ -0,0 +1,174 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+
+  <!-- bootstrap link -->
+
+  <link href= "https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css" rel="stylesheet"
+    integrity="sha384-EVSTQN3/azprG1Anm3QDgpJLIm9Nao0Yz1ztcQTwFspd3yD65VohhpuuCOmLASjC" crossorigin="anonymous">
+
+  <!-- font awesome -->
+
+  <link rel="stylesheet" href= "https://pro.fontawesome.com/releases/v5.10.0/css/all.css" integrity="sha384-AYmEC3Yw5cVb3ZcuHtOA93w35dYTsvhLPVnYs9eStHfGJvOvKxVfELGroGkvsg+p
+    " crossorigin="anonymous" />
+  <title>DDT </title>
+  <link rel="icon" href= "./asset/image/logo_mobile.png">
+  <link rel="stylesheet" href= "./asset/scss-about/about.css">
+  <!-- <link rel="stylesheet" href= "./css/style.css"> -->
+</head>
+
+<body>
+  <header class= "contain-elements">
+
+    <nav>
+      <!-- header and nav for mobile -->
```

Comment:
- [ ] **As the previous reviewer pointed out, your navbar is still overflowed. Please fix it**

![image](https://user-images.githubusercontent.com/59168430/124353603-20f8a980-dc25-11eb-866a-9d40c47b1a14.png)

As you can see in the image above, the navbar is overflowing out of the main layout. Please fix this.